.buildkite/pipeline.merge.yml

```diff
@@ -26,6 +25,23 @@ steps:
     artifact_paths:
       - "*.grapl-artifacts.json"
 
+  - label: ":linux: Build non-Docker packages"
```

Comment:
I'd call this `":firecracker: Build Firecracker packages"`; there's an emoji, we might as well use it 😂

Reply:
done